packages/playground/_stories/fiori/Page/Page.stories.ts

```diff
@@ -23,58 +25,59 @@ export default {
     argTypes,
 } as Meta<Page>;
 
-const Template: UI5StoryArgs<Page, StoryArgsSlots> = (args) => html`<div></div>`;
+const Template: UI5StoryArgs<Page, StoryArgsSlots> = (args) => {
+    return html`
+<ui5-page
+    id="page"
```

Comment:
By defining the ID like that, doesn't this mean that any ui5-page that we define will have id="page"?